src/targets/ref/include/migraphx/ref/context.hpp

```diff
@@ -25,13 +25,16 @@
 #define MIGRAPHX_GUARD_RTGLIB_CONTEXT_HPP
 
 #include <migraphx/config.hpp>
+#include <migraphx/any_ptr.hpp>
 
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 namespace ref {
 
 struct context
 {
+    void wait_for(any_ptr) const {}
+    void finish_on(any_ptr) const {}
```

Comment:
These methods shouldn't need to be added since we default the methods(with `default = 'wait_for_context'`).

Reply:
Okay I can remove these